docs/tutorials/docker.rst

```diff
@@ -1,12 +1,13 @@
 Run using docker-compose
 =========================================
-Execute the following command the deployment will be started using docker-compose. This will build the dockerimage
+Execute the following commands to start the deployment using docker-compose. This will build the docker image
 containing the application and creates all required containers including the database and the message queue.
 
 Note that you might need to authorize with an Access Token in order to access the qunicorn package on GitHub.
```

Comment:
Mark it clearly as optional as it is not necessary at the moment and probably wont be

Reply:
We can just remove the entire note as I believe it is no longer required.